src/App.vue

```diff
@@ -171,8 +172,16 @@ export default {
   },
 
   computed: {
-    ...mapGetters(['getIdentifier', 'getEntityName', 'getEntityType', 'isRoleStaff',
-      'getBusinessUrl', 'getCreateUrl', 'getAuthApiUrl', 'getRegHomeUrl']),
+    ...mapGetters([
+      'getAuthApiUrl',
+      'getBusinessUrl',
+      'getCreateUrl',
+      'getEntityName',
+      'getEntityType',
+      'getIdentifier',
+      'getRegHomeUrl',
+      'isRoleStaff'
+    ]),
```

Comment:
Sorted and reformatted for easier finding of getters.

Ditto below for actions and mutations.